src/java/components/hu.bme.mit.inf.modes3.components.dashboard/src/main/webapp/public_html/js/config.js

```diff
@@ -278,102 +248,108 @@ var settings = {
 		"t06", "t06-div", "t06-str"
 	],
 	locomotives : [
-	
-	//ide sok vonat
```

Comment:
🇭🇺  comment

Reply:
This comment leaved mistakenly after a consulting with Edina Pap, will be removed.